src/dune_rules/mdx.ml

```diff
@@ -201,26 +226,43 @@ let gen_rules_for_single_file stanza ~sctx ~dir ~expander ~mdx_prog src =
     (* Add the rule for generating the .corrected file using ocaml-mdx test *)
     let mdx_action =
       let open Action_builder.With_targets.O in
+      (* Test files dependencies*)
```

Comment:
I think you can get rid of these comments by naming the variables appropriately, e.g. `let mdx_input_dependencies =`, etc, but that's a matter of taste.